### DISCUSSION TITLE
Pid autotune

### DISCUSSION THREAD

**Discussion by pipoJobje** [Jyers/Marlin#281] [Q&A] [answered]
Hello, so I have just installed your firmware because I heard many great stories about it. So far it is great! I hope you realise how much this means for a lot of people. 
I have one question though. I noticed that my temps are fluctuating a lot more than before, does this mean I need to do an Pid-autotune? If so, can I just do it with pronterface or do I need something else?

**Comment by Jyers** [comment, marked as answer]
The PID settings are the same as the default ender 2 v2 configs, so I don't know why it would be fluctuating more. Either way you can definitely do a pid autotune from proterface. Give this a read for the required commands: https://marlinfw.org/docs/gcode/M303.html
I'm also hoping to implement pid autotune options in the on screen menu's as well, so if you ever need to do it again keep an eye out for that addition.
